platforms/afu_top_ifc_db/README_AFU

```diff
@@ -23,29 +21,44 @@ field:
 
 
 In addition to the name, some module-arguments parameters may be
-overridden in the AFU JSON database.  For example, the following makes
-local memory optional:
+overridden in the AFU JSON database.  For example, the following
+requests automatic registering of CCI-P signals on platforms that
+need more than one register stage, makes local memory optional and
```

Comment:
"makes local memory optional" -> does this mean that the interface isn't exposed to the AFU developer? or does it mean the interface is exposed, but using it doesn't have any effect?

Reply:
When the AFU says memory is optional, the AFU must be prepared to deal with the local_mem interface maybe being there and maybe not. There are preprocessor macros generated that will tell it what happened.